BainaRocio/project/src/com/project/marketordering/market/Mercado.java

```diff
@@ -0,0 +1,42 @@
+package com.project.marketordering.market;
+
+import com.project.marketordering.person.Empleado;
+import com.project.marketordering.products.Product;
+
+import java.util.List;
+
+public class Mercado {
+    private List<Empleado> empleadoList;
+    private List<PurchaseCar> purchaseCarList;
+    private List<Product> productList;
+
+    public Mercado(List<Empleado> empleadoList, List<PurchaseCar> purchaseCarList, List<Product> productList) {
```

Comment:
When working with collections is not a great idea to pass them as parameters because copying collections can cause errors and performance issues.
I'll suggest to add methods that will wrap the use of your collection, for example:
public String addEmployee(Employee employee) {
this.employeeList.add(employee);
}
You can do this so in the constructor you will not receive a list as a parameter and you will only initialize your list:
this.employeeList= new ArrayList<>();